### ISSUE TITLE
iOS fat binary creation does not work.

### ISSUE THREAD

**Issue by freemanfromgodotengine** [godotengine/godot#7329]
**Operating system or device - Godot version:**
OS X Yosemite 10.10.5,  Godot 2.1-stable, Xcode 7.2.

**Issue description** (what happened, and what was expected):
I am not able to create fat binary for iOS.
In docs http://docs.godotengine.org/en/stable/reference/compiling_for_ios.html there is an instruction

> $ scons p=iphone tools=no bits=32 target=release
> $ scons p=iphone tools=no bits=64 target=release
> $ lipo -create bin/godot.iphone.opt.32 **arm64** bin/godot.iphone.opt.64 -output bin/godot.iphone.opt.universal

So I renamed the godot.iphone.opt.arm files I got after compilation accordingly, but lipo errors with
fatal error: 

> /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo: can't open input file: arm64 (No such file or directory)

When I try to correct the typo and add -arch before arm64, to indicate the architecture of the input file:
`lipo -create bin/godot.iphone.opt.32 -arch arm64 bin/godot.iphone.opt.64 -output ./bin/godot.iphone.opt.universal`

>
I get error: 
> lipo: specifed architecture type (arm64) for file (./godot.iphone.opt.64) does not match its cputype (12) and cpusubtype (9) (should be cputype (16777228) and cpusubtype (0))

**UPDATE**: I additionally checked `lipo -info godot.iphone.opt.64` and it shows that it's armv7 architecture?! I am sure I did compile it with bits=64 option.

I also tried -arch arm64v8 and that ended with similar error.

I assume there is an error in docs, or there is a bug with compiling 64 bit version of iphone template.

I would appreciate if someone push me in the right direction how to compile fat binary for iOS.

**Comment by akien-mga** [comment]
That's likely an issue with the docs. Given https://github.com/GodotBuilder/godot-builds/blob/master/.travis.yml, the correct process is:
```
scons platform=iphone arch=arm target=release
scons platform=iphone arch=arm64 target=release
lipo -create -arch armv7 bin/godot.iphone.opt.arm -arch arm64 bin/godot.iphone.opt.arm64 -output bin/godot.iphone.opt.universal
```

**Comment by Noshyaar** [comment]
First of all thank you for your report and sorry for the delay.

We released Godot 3.0 in January 2018 after 18 months of work, fixing many old issues either directly, or by obsoleting/replacing the features they were referring to.

We still have hundreds of issues whose relevance/reproducibility needs to be checked against the [current stable version](https://godotengine.org/download), and that's where you can help us.
**Could you check if the issue that you described initially is still relevant/reproducible in Godot 3.0 or any newer version, and comment about it here?**

For bug reports, please also make sure that the issue contains detailed steps to reproduce the bug and, if possible, a zipped project that can be used to reproduce it right away. This greatly speeds up debugging and bugfixing tasks for our contributors.

Our Bugsquad will review this issue more in-depth in 15 days, and potentially close it if its relevance could not be confirmed.

Thanks in advance.

*Note: This message is being copy-pasted to many "stale" issues (90+ days without activity). It might happen that it is not meaningful for this specific issue or appears oblivious of the issue's context, if so please comment to notify the Bugsquad about it.*

**Comment by akien-mga** [comment]
> Our Bugsquad will review this issue more in-depth in 15 days, and potentially close it if its relevance could not be confirmed.

As there was no update since the previous post, we close this issue.

If it is still relevant in the way it was described and discussed above, please comment to ask for it to be reevaluated. If it is only partly relevant, or if the original issue has changed, it would be better to open a new issue (potentially referring to this one) focused on the current state.
